data/competencies.yml

```diff
@@ -1,16 +1,292 @@
+  summary: Writes pull requests that are helpful both to the reviewer and those debugging the the code later
+  examples:
+    - Writes PR descriptions that explain why the PR is needed
+    - Links to relevant Jira tickets in PRs
+    - Adds screenshots to show changes if applicable
+    - Provides context that they learned when researching a PR
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: 57c50830-f2ef-4c61-97fa-ec80b0624473
+  summary: Regularly reads peers' PRs and asks questions in order to learn
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: 5d637365-7e38-432d-bcec-8652f06d8109
+  summary: Remembers to write more straightforward tickets and bug reports to track the work expected of them, perhaps with prompting and guidance
+  examples:
+    - Checks there is a corresponding ticket or bug report before starting any work, and flags it if not
+    - Writes reasonable "steps to reproduce" in a reasonably straightforward bug report when prompted
+    - Adds relevant links and screenshots to pages affected by a bug when prompted
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: dcfbf24d-04e2-4029-88a0-93479e4d04a8
+  summary: Receives feedback and acts upon it
+  examples:
+    - Implements requested changes on a pull request
+    - Responds to comments on a Google document
+    - If they receive feedback from the team that sometimes they don't seem well-prepared for morning stand-ups, takes 5 minutes before stand-ups to gather thoughts about what the status of their work is
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: c8d62965-dd71-4a90-83df-a4c924a40fd8
+  summary: Is able to recognise when they are having difficulties, provides information so that other people can help
+  examples:
+    - Recognises personal work-related problems such as falling behind, not feeling confident, or feeling overwhelmed
+    - Communicates these problems rapidly to their line manager, mentor or tech lead so that help and guidance can be provided
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: da526878-694a-4a73-b678-f3e46301e1b6
+  summary: Presents their own work clearly to a product owner or tech lead
+  examples:
+    - Talks about their progress in standup without going into deep technical detail
+    - Explains their approach to a technical problem to their tech lead
+    - Gives a live demo of a feature that they worked on to their team
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: b8bd799d-6712-4882-b1a7-4f66166c3315
+  summary: Effectively communicates when they are unable to attend expected events
+  examples:
+    - Informs meeting organisers of meetings they cannot attend
```

Comment:
Too many examples for what is one of the simpler competencies to meet?

Reply:
Unfortunately, these have come from real life situations, which is why they're all there. But, yes, it's all quite verbose, and I'll tighten those up. Thanks.